go/cmd/vtgate/vtgate.go

```diff
@@ -40,12 +41,16 @@ import (
 )
 
 var (
-	cell                     = flag.String("cell", "test_nj", "cell to use")
-	tabletTypesToWait        = flag.String("tablet_types_to_wait", "", "wait till connected for specified tablet types during Gateway initialization")
-	plannerVersion           = flag.String("planner-version", "", "Sets the default planner to use when the session has not changed it. Valid values are: V3, Gen4, Gen4Greedy and Gen4Fallback. Gen4Fallback tries the gen4 planner and falls back to the V3 planner if the gen4 fails.")
-	plannerVersionDeprecated = flag.String("planner_version", "", "Deprecated flag. Use planner-version instead")
+	cell, tabletTypesToWait, plannerVersion, plannerVersionDeprecated string
 )
 
+func RegisterFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&cell, "cell", "test_nj", "cell to use")
```

Comment:
Instead of adding a default here, we are following a different pattern, where the default lives with variable declaration

For example, 
```golang
var cell = "test_nj"
fs.StringVar(&cell, "cell", cell, "cell to use")
```

Reply:
The only problem with that is if I want to declare variable as var cell *string then it would not be possible to do it.

Reply:
we should follow @GuptaManan100's suggestion, i'm not sure why you want to declare as `*string` specifically (especially since in this specific instance `cell` is type `string`)